csharp/src/Ice/Communicator-PluginManager.cs

```diff
@@ -26,15 +25,15 @@ public sealed partial class Communicator
         private static readonly List<string> _loadOnInitialization = new List<string>();
         private static readonly Dictionary<string, IPluginFactory> _pluginFactories =
             new Dictionary<string, IPluginFactory>();
-        private bool _pluginsInitialized;
+
         private readonly List<(string Name, IPlugin Plugin)> _plugins = new List<(string Name, IPlugin Plugin)>();
```

Comment:
We chould use `new()` here for the init of the data members.

Reply:
Fixed.